source/_integrations/airly.markdown

```diff
@@ -15,15 +15,13 @@ ha_platforms:
   - sensor
 ---
 
-The `airly` integration uses the [Airly](https://airly.eu/) web service as a source for air quality data for your location.
+The Airly integration uses the [Airly](https://airly.eu/) web service as a source for air quality data for your location.
 
 ## Setup
 
 To generate an Airly API key, go to [Airly for developers](https://developer.airly.eu/register) page.
 
-## Configuration
-
-To add Airly to your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Airly**. By default, the values will be taken from the Home Assistant configuration.
+{% include integrations/config_flow.md %}
```

Comment:
This an example on why I think having the header in that template, makes this a bit more unclear in Markdown.

Reply:
ah, right...

```
{% if include.hide_header != true %}

## Configuration

{% endif %}
```

And then:
`{% include integrations/config_flow.md hide_header=true %}`